src/components/shared/Select/index.tsx

```diff
@@ -253,6 +255,8 @@ interface ItemStyle {
 interface Props {
   testID?: string;
   theme?: ThemeEnum;
+  title: string;
```

Comment:
Title value should be used as optional rather than required.
```suggestion
  title?: string;
```